code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -22,6 +22,11 @@
 	channels = list("Syndicate" = 1)
 	origin_tech = "syndicate=1;engineering=3;bluespace=2"
 	syndie = 1 //Signifies that it de-crypts Syndicate transmissions
+	var/fake_name = "Agent ALERT_A_CODER"
```

Comment:
There's a man who leads a life of danger

To everyone he meets he stays a stranger...